types/Img.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace plugin\struct\types;
+
+use plugin\struct\meta\ValidationException;
+
+class Img extends AbstractBaseType {
+
+    protected $config = array(
+    );
+
+    /**
+     * Output the stored data
+     *
+     * @param string|int $value the value stored in the database
+     * @param \Doku_Renderer $R the renderer currently used to render the data
+     * @param string $mode The mode the output is rendered in (eg. XHTML)
+     * @return bool true if $mode could be satisfied
+     */
+    public function renderValue($value, \Doku_Renderer $R, $mode) {
+        if (!empty($value)) {
```

Comment:
I believe we do not render empty values? We should set up some rules on what types do not need to check themselves (like validate() is never called for blank fields).